cmd/minikube/cmd/root.go

```diff
@@ -40,6 +40,8 @@ import (
 	"k8s.io/minikube/pkg/minikube/translate"
 )
 
+const defaultClusterBootstrapper = "kubeadm"
```

Comment:
no need to define this as a const: just inline it into the line where it is used (`RootCmd.PersistentFlags().StringP(...)`)